### PR TITLE
Contact form elements and styling

### PR DESCRIPTION
**Pull request by herokudev**
Contact form elements and styling.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,5 +124,27 @@ <h3 class="about-me-title">About Me</h3>
...
         </section>
+        <form class="my-contact-form" action="https://formspree.io/f/mayadawe" method="POST">
+            <h1 class="contact-form-header">I'm always <br>interested in hearing <br>about new projects, <br>so if you'd like to <br>chat please get in</h1>
+            <input class="contact-form-name" type="text" id="name" name="user_name" maxlength="30" placeholder="Full name" required>
+            <img class="field-separator" src="images/Container.png" alt="">
+            <input class="contact-form-email" type="email" id="email" name="user_email" placeholder="Email address" required>
+            <img class="field-separator" src="images/Container.png" alt="">
+            <textarea class="contact-form-message" name="Message" maxlength="500" placeholder="Enter text here" required></textarea>
+            <button type="submit" class="button button4">Get in Touch</button>
+        </form>
```
Comment (Emmyn5600) [inline index.html:135]: - [ ] Good to validate your fields however, there is some issue to be solved, I have tried to submit a message with the wrong email format and it works, please fix this issue in different two ways:
  * Include a pattern  used to validate email in your HTML or
  * Make sure that the formspree show us an error after sending a message with a wrong email.
  
form1

File: index.html [hunk 2]
```diff
@@ -124,5 +124,27 @@ <h3 class="about-me-title">About Me</h3>
...
+        <img class="field-separator" src="images/Rectangle-42.png" alt="">
+        <section class="box-box5">
+            <ul class="social-nav-list">
+                <li class="social-nav-item"><img src="images/bottom-GitHub.png" alt=""></li>
+                <li class="social-nav-item"><img src="images/bottom-Linkedin.png" alt=""></li>
+                <li class="social-nav-item"><img src="images/bottom-Angelist.png" alt=""></li>
+                <li class="social-nav-item"><img src="images/bottom-Twitter.png" alt=""></li>
+                <li class="social-nav-item"><img src="images/bottom-Medium.png" alt=""></li>
+        </ul>  
+        </section>
```
Comment (Emmyn5600) [inline index.html:145]: - [ ] Nice to add the social media icons properly, so for the user experience (UX), I suggest you to make cursor as pointer in your css.

File: css/style.css [hunk 3]
```diff
@@ -282,3 +282,75 @@ main {
...
+.my-contact-form {
+  display: flex;
+  flex-direction: column;
+  padding: 80px 24px 80px 24px;
+}
+
+.contact-form-header {
+  font-family: "Inter", Arial, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 32px;
```
Comment (Emmyn5600) [inline css/style.css:295]: - [ ] Kindly, fix this issue by changing the font-size of your text
form2

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
## STATUS: Changes Requested :recycle:

Hi @herokudev 

Good job so far!:clap::clap:
You have done well in implementing the project requirements :clap:, However, there are some issues that you still need to work on to go to the next project but you are almost there!:muscle:

##  Required Changes:writing_hand:
  
**_Check the comments under the review._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: css/style.css (hunk 3 above)
Comment (herokudev) [reply css/style.css:295]: OK, done!
I change the font-family to the my-contact-form class.

File: index.html (hunk 2 above)
Comment (herokudev) [reply index.html:145]: OK, done! 
Change completed with the "cursor: pointer" property.

File: index.html (hunk 1 above)
Comment (herokudev) [reply index.html:135]: OK, done!
Pattern added.

**Review by mimipeshy — APPROVED** [review]
## Hi @herokudev 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by herokudev** [comment]
Merging approved changes for contact form.

**Pull request merged**